spec/std/base64_spec.cr

```diff
@@ -63,20 +75,16 @@ describe "Base64" do
         "Now is the time for all good coders\nto learn Crystal")
     end
 
-    it "encode to stream" do
+    it "encode to stream returns number of written characters" do
       io = IO::Memory.new
       count = Base64.encode("Now is the time for all good coders\nto learn Crystal", io)
       count.should eq 74
-      io.rewind
-      io.gets_to_end.should eq "Tm93IGlzIHRoZSB0aW1lIGZvciBhbGwgZ29vZCBjb2RlcnMKdG8gbGVhcm4g\nQ3J5c3RhbA==\n"
```

Comment:
why you remove this line?

Reply:
Covered by `assert_prints base64_encode(a), b` above (if the `IO`-accepting overload doesn't return anything we could drop this test case entirely).